src/llmcompressor/transformers/utils/helpers.py

```diff
@@ -54,3 +59,101 @@ def detect_last_checkpoint(
             )
 
     return last_checkpoint
+
+
+def is_model_quantized_from_path(path: str) -> bool:
+    """
+    Determine if model from path is quantized based
+    on the config
+
+    :param path: path to the model or HF stub
+    :return: True if config contains quantization_config from the given path
+
+    """
+    config = AutoConfig.from_pretrained(path)
```

Comment:
nit: check if also compressed-tensors as all quant models will have this quantization_config

Reply:
done!